### PR TITLE
[Merged by Bors] - Add mainnet merge values 🐼

### PR DESCRIPTION
**Pull request by paulhauner**
## Issue Addressed

NA

## Proposed Changes

Adds **tentative** values for the merge TTD and Bellatrix as per https://github.com/ethereum/consensus-specs/pull/2969

## Additional Info

- ~~Blocked on https://github.com/ethereum/consensus-specs/pull/2969~~

### CODE REVIEW

**Comment by paulhauner** [comment]
We're going to wait for the ACD call on 2022-08-18 before we cut a release with these parameters, but considering that https://github.com/ethereum/consensus-specs/pull/2969 is merged I think it's a good time for us to get this into `unstable` and start testing it.

**Review by michaelsproul — APPROVED** [review]
All aboard the merge train :panda_face: :bullettrain_side:

**Comment by michaelsproul** [comment]
@paulhauner we need to update the hardcoded values in `ChainSpec::mainnet`

**Comment by paulhauner** [comment]
> @paulhauner we need to update the hardcoded values in `ChainSpec::mainnet`

Fixed! I had to fix some other tests that were assuming an Altair epoch that was higher than the Bellatrix epoch, too.

**Comment by paulhauner** [comment]
I'm going to try adding these tests now: https://github.com/ethereum/consensus-specs/releases/tag/v1.2.0-rc.2

**Comment by paulhauner** [comment]
I had to make sure that the `minimal` spec still uses the placeholder TTD: https://github.com/sigp/lighthouse/pull/3462/commits/f465785c5606cf99de0b5d70aea1793a99bceb50

**Comment by paulhauner** [comment]
... and ignore some new tests: https://github.com/sigp/lighthouse/pull/3462/commits/c19ab1f8b599be4967c11e0398c34fd02bf01e38

File: testing/ef_tests/check_all_files_accessed.py [hunk 1]
```diff
@@ -25,12 +25,20 @@
     # Intentionally omitted, as per https://github.com/sigp/lighthouse/issues/1835
     "tests/.*/.*/ssz_static/Eth1Block/",
     "tests/.*/.*/ssz_static/PowBlock/",
+    # light_clinet
```
Comment (michaelsproul) [inline testing/ef_tests/check_all_files_accessed.py]:
```suggestion
    # light_client
```

**Review by michaelsproul — APPROVED** [review]
Looks good modulo one typo

**Pull request closed without merging**